src/avro_json_decoder.erl

```diff
@@ -47,37 +49,44 @@ decode_schema(Json) ->
 decode_schema(JsonSchema, ExtractTypeFun) ->
   parse_schema(mochijson3:decode(JsonSchema), "", ExtractTypeFun).
 
-%% Decode value specified as Json string according to Avro schema
-%% in Schema. ExtractTypeFun should be provided to retrieve types
-%% specified by their names inside Schema.
--spec decode_value(string(),
-                   avro_type_or_name(),
-                   fun((string()) -> avro_type()))
-                  -> avro_value().
-
-decode_value(JsonValue, Schema, ExtractTypeFun) ->
-  parse_value(mochijson3:decode(JsonValue), Schema, ExtractTypeFun).
-
-%% Experimental support for jsonx decoding.
 %% jsonx is about 12 times faster than mochijson3 and almost compatible
 %% with it. The one discovered incompatibility is that jsonx performs more
 %% strict checks on incoming json strings and can fail on some cases
 %% (for example, on non-ascii symbols) while mochijson3 can parse such strings
 %% without problems.
 %% Current strategy is to use jsonx as the main parser and fall back to
```

Comment:
Doesn't look like jsonx is the main parser, one should explicitly specify it in Options.
May be use `{decoder, jsonx}` in decode_value/3?